src/dlls/mscordac/CMakeLists.txt

```diff
@@ -184,4 +184,21 @@ endif(WIN32)
 target_link_libraries(mscordaccore PRIVATE ${COREDAC_LIBRARIES})
 
 # add the install targets
-install_clr(mscordaccore)
+install_clr(TARGETS mscordaccore)
+install_clr(TARGETS mscordaccore DESTINATION sharedFramework)
+if(WIN32)
+    set(LONG_NAME_HOST_ARCH ${CLR_CMAKE_HOST_ARCH})
+    if (CLR_CMAKE_PLATFORM_ARCH_AMD64)
+        set(LONG_NAME_HOST_ARCH "amd64")
+    endif()
+
+    set(LONG_NAME_TARGET_ARCH ${CLR_CMAKE_TARGET_ARCH})
+    if (CLR_CMAKE_PLATFORM_ARCH_AMD64)
+        set(LONG_NAME_TARGET_ARCH "amd64")
+    endif()
+    file(READ "${CLR_DIR}/bin/obj/_version.h" NATIVE_VERSION_HEADER)
```

Comment:
What happens in CMake if the `file` function fails to open? (e.g. files doesn't exist). Perhaps we should have a bit of context as to why this needs to happen and maybe a custom error message that indicates what has gone wrong rather than a possible silent error or the equally annoying "Failed to open file ...", both of which don't really help users know how to unblock themselves.

Reply:
If it fails to open, it will error out. I'll add in a `message()` command beforehand explaining what this step is doing before the `file` command so a failure here could be actionable.